### PR TITLE
Add README.md to grpc folder

### PR DESCRIPTION
**Pull request by dr-venkman**
Add README to external/grpc folder. This document covers details from how to build gRPC module on TizenRT, as well as develop, build and execute TizenRT applications that use gRPC.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/grpc/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# gRPC framework for TizenRT.
```
Comment (sunghan-chang) [inline external/grpc/README.md]: Let's remove '.' and do we need to say "for TizenRT"? Here is only TizenRT.

File: external/grpc/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+TizenRT features the gRPC framework from the opensource project [gRPC](https://github.com/grpc/grpc), and is versioned at 1.9.
+TizenRT's version of gRPC supports only C++ binding, so it is expected that the developer should write C++ applications in order to
```
Comment (sunghan-chang) [inline external/grpc/README.md]: As you wrote, let's change a line for readablility. Add 2 spaces at end of previous line.

File: external/grpc/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+TizenRT features the gRPC framework from the opensource project [gRPC](https://github.com/grpc/grpc), and is versioned at 1.9.
+TizenRT's version of gRPC supports only C++ binding, so it is expected that the developer should write C++ applications in order to 
+use gRPC. In this document, we cover two main aspects of porting gRPC to TizenRT, mainly details on how to build gRPC library, and how
+to develop, build and run applications that use gRPC.
+
+## Build gRPC in TizenRT
+The source files for gRPC are located at `external/grpc`, and the associated header files are located at 
+`external/include/grpc` folders. To build gRPC in TizenRT, you need to enable CONFIG_GRPC defined in `external/grpc/Kconfig.protocol`, as shown below:
+```
+config GRPC
```
Comment (sunghan-chang) [inline external/grpc/README.md]: You had better give the structure on menuconfig, not source code level of Kconfig.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/grpc/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+setenv("GRPC_VERBOSITY", "DEBUG", 1);
+```
+gRPC also allows logging tracers by setting the `GRPC_TRACE` variable to `all`. However, this functionality is not enabled yet, so interested developers must
+manually enable the tracers in the gRPC source.
+
+
+
+
+
+
```
Comment (sunghan-chang) [inline external/grpc/README.md]: Could you remove meanless empty lines?

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**